experimental/LieAlgebras/src/AbstractRootSystem.jl

```diff
@@ -145,16 +145,16 @@ function Base.:(*)(n::IntegerUnion, w::WeightLatticeElem)
   return WeightLatticeElem(w.root_system, n * w.vec)
 end
 
-function Base.:(+)(r::WeightLatticeElem, s::WeightLatticeElem)
-  @req r.parent === s.parent "r and s must belong to the same root lattice"
+function Base.:(+)(w::WeightLatticeElem, w2::WeightLatticeElem)
+  @req w.root_system === w2.root_system "$v and $w must belong to the same root lattice"
```

Comment:
`$v` does not exist here. 

Below, you only use `w` instead of `$w`. It would be great to have that consistently